src/components/ProfileEditEmailPage.js

```diff
@@ -43,25 +43,42 @@ const useStyles = makeStyles((theme) => ({
 const ProfileEditEmailPage = () => {
   const classes = useStyles()
   const { updateUser } = useAuthActions()
+  const [error, setError] = useState(false)
+  const [status, setStatus] = useState("idle")
   const { user } = useAuthState()
   const currentEmail = user.email
 
-  const createEmail = (event) => {
-    // 1. Prevent that form from naughtily self-submitting
+  const createEmail = async (event) => {
+    // P.L.A.N. Ruby's Rescue Mission
```

Comment:
Please remove these comments, other than that it looks good. You may add comments, but they should be understandable by other devs.